ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -0,0 +1,278 @@
+# 'geometry_type' specifies the geometry type for this layer
+# 'layer_name' is the name you wish to be used for this shapefile layer in postgres
+dbKey: geoid10
+dbKeyDataType: string
+shape_projection: "EPSG:4326"
+geometry_type: MultiPolygon
+layer_name: seattle_blkgrpce10
+
+# The variables below define the aggregation name and table name for aggregate statistics
+aggregation_name: "by_census_block"
+aggregation_stats_table_name: "block_statistics"
+
+# Define the latitude and longitude for the southwest and northeast corners of a bounding 
+# box, defining the region where you are interested in pulling M-Lab NDT data.
+map_coordinate_southwest: 
+  lat: 47.3630134401
+  lon: -122.6733398438
+
+map_coordinate_northeast: 
+  lat: 47.8076208172
+  lon: -121.9509887695
+
+# The four variable below are derived from your bounding box coordinates above.
+map_bounding_box: "{{ map_coordinate_southwest.lon }},{{ map_coordinate_southwest.lat }},{{ map_coordinate_northeast.lon }},{{ map_coordinate_northeast.lat }}"
+
+# The two dates below define the date range for M-Lab NDT test data. In production,
+# it is advisable to set 'map_data_data_end_date' to a date well in the future, for
+# example: "Dec 31 2050 00:00:00"
+map_data_start_date: "Jan 1 2017 00:00:00"
+map_data_end_date: "Feb 15 2017 00:00:00"
+
+###
+# WEBSITE MAP INFORMATION
+#
+# The section below defines variables to be used on the website map.
+
+# Piecewise requires a topojson file for the website to use. This topojson file must
+# be created from the shapefile you defined above. 
+# Save your topojson file into the folder /local_customizations/geofiles
+# and then enter its filename here.
+topojson_file: seattle_census10_blockgroups.topojson
+
+# 'geoKey' is the field name in your topojson file which corresponds to the unique key 
+# in your shapefile, and is typically the same as dbKey, but in CAPITAL LETTERS.
+geoKey: GEOID10
+
+# 'mapscript_start_year' is the first year you are interested in showing in the year dropdown
+# 'mapscript_start_month' lets you define the order of months in the UI.
+mapscript_start_year: 2016
+mapscript_start_month: 1
+
+# 'mapscript_base_layer' defines the type of base layer to be used on the map.
+# Either Mapbox.com or OpenStreetMaps base layers are supported. 
+#
+# Set mapscript_base_layer to either: 'mapbox' OR 'osm'
+mapscript_base_layer: osm
+
+# If you would like to customize the OSM base layer API url, for example if you
+# are hosting custom maps and tiles on your own tile server, update the variable below.
+# The default OSM url is provided.
+mapscript_osm_api_url: "http://{s}.tile.osm.org/{z}/{x}/{y}.png"
+
+# If you are using a Mapbox base layer, set the API URL for your Mapbox tiles here.
+mapscript_mapbox_api_url: "https://api.mapbox.com/styles/v1/newamerica/ciy31olzw00792sp6w98b76i0/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoibmV3YW1lcmljYSIsImEiOiIyM3ZnYUtrIn0.57fFgg_iM7S1wLH2GQC71g"
```

Comment:
since we aren't using mapbox here, I think it'd be best to not check-in an access-key to the repo